Gemfile

```diff
@@ -67,3 +67,6 @@ end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
+
+# https://stackoverflow.com/questions/66919504/your-bundle-is-locked-to-mimemagic-0-3-5-but-that-version-could-not-be-found
+gem 'mimemagic', github: 'mimemagicrb/mimemagic', ref: '01f92d86d15d85cfd0f20dabd025dcbd36a8a60f'
```

Comment:
@JackBlackLight Could you try updating to Rails 6.0.3.7 instead of pinning mimemagic to this github ref?  The mimemagic yanked gem lock issue was supposedly fixed in 6.0.3.6 (or later).